lib/mix/lib/mix/tasks/format.ex

```diff
@@ -22,6 +22,10 @@ defmodule Mix.Tasks.Format do
     * `:inputs` (a list of paths and patterns) - specifies the default inputs
       to be used by this task. For example, `["mix.exs", "{config,lib,test}/**/*.{ex,exs}"]`.
 
+    * `:subdirectories` (a list of paths and patterns) - specifies subdirectories
```

Comment:
> Overall it looks good. As far as I can tell, we don't merge the option in `a/b/.formatter.exs` with the options in `a/.formatter.exs` when formatting `a/b/foo.ex`, right? If I got it correctly and it is like this...

I also had the same doubts, and I had to look at the source code. I think a more detailed explanation about the inheritance rules between `.formatter.exs` and its `:subdirectories` could be helpful :)

Reply:
I have clarified it here, thank you!